src/main.rs

```diff
@@ -858,6 +859,21 @@ mod tests {
         Parser::parse(contents.into().as_bytes()).unwrap()
     }
 
+    // Parse a string of code that is meant to fail
+    fn parse_str_fail(contents: impl Into<String>) -> Parser {
```

Comment:
The `parse_str` function has the `-> Parser` as response because we can use that later on to verify the parsed content we don't need that here.  
```suggestion
    fn parse_str_fail(contents: impl Into<String>) {
```